### DISCUSSION TITLE
Is it possible to establish a communication group

### DISCUSSION THREAD

**Discussion by wavemomo** [smartxworks/virtink#57] [General]
As the title said, can we establish a communication group  to make the discussion more efficient?

**Comment by fengye87** [comment]
We have a WeChat group, feel free to join in. We may open a slack channel in the future, but for now WeChat group is what we have.

![WechatIMG121](https://user-images.githubusercontent.com/7160387/191636132-5dfba6f0-fcb2-45bd-82f0-6ef8e115d6d8.jpeg)

**Reply by wavemomo** [reply]
Awesome, thanks for your efforts.
